Transliterate.cs

```diff
@@ -1,44 +1,64 @@
+// This sample requires C# 7.1 or later for async/await.
+
 using System;
 using System.Net.Http;
 using System.Text;
+using System.Threading.Tasks;
+// Install Newtonsoft.Json with NuGet
 using Newtonsoft.Json;
 
-namespace TransliterateText
+namespace TransliterateTextSample
 {
+    /// <summary>
+    /// The C# classes that represents the JSON returned by the Translator Text API.
+    /// </summary>
+    public class TransliterationResult
+    {
+        public string Text { get; set; }
+        public string Script { get; set; }
+    }
+
     class Program
     {
-        static void TransliterateText()
+        // Async call to the Translator Text API
+        static public async Task TransliterateTextRequest(string subscriptionKey, string host, string route, string inputText)
         {
-            string host = "https://api.cognitive.microsofttranslator.com";
-            string route = "/transliterate?api-version=3.0&language=ja&fromScript=jpan&toScript=latn";
-            string subscriptionKey = "YOUR_SUBSCRIPTION_KEY";
-
-            System.Object[] body = new System.Object[] { new { Text = @"こんにちは" } };
```

Comment:
I think you can just do new object[] (lower case o and no System.)

Reply:
Good suggestion. Just tested and works great.